katalog/fluentd/deploy.yml

```diff
@@ -18,7 +18,8 @@ spec:
       - operator: Exists
       containers:
       - name: fluentd
-        image: sighup/fluentd-elasticsearch:1.2.8
+        image: quay.io/sighup/fluentd-elasticsearch:v1.10.2-debian-1.0
```

Comment:
maybe we could consistently set the image tag from the kustomization.yaml as in previous deployments in this pr.

Reply:
found it is already set, so just remove it from here to avoid confusion